docs/configuration/service/ipoe-server.rst

```diff
@@ -52,7 +52,7 @@ the configuration.
 
 .. cfgcmd:: set service ipoe-server authentication interface <interface> mac <MAC>
 
-    Creates local pppoe user with username=**<interface>** and
+    Creates local ipoe user with username=**<interface>** and
```

Comment:
There should be `IPoE` ie abbr, the same for `PPPoE` or `L2TP`

Reply:
done